gunicorn/arbiter.py

```diff
@@ -494,6 +496,11 @@ def manage_workers(self):
 
     def spawn_worker(self):
         self.worker_age += 1
+        worker_id = -1
+        for i in range(self.num_workers):
```

Comment:
If just using the PID to identify, adding this feature would be unnecessary, as the instances launched by gunicorn can already use their PID as a unique identifier. Also, if the hash uses worker.age, they would change when a worker is reaped and spawned, as worker.age continues to increment as long as gunicorn is running. This would generate a number of hashes that is greater than the number of workers, which the current implementation avoids. I still believe using a basic counter covers the most use cases and keeps things relatively simple.

One alternative that may satisfy us both would be adding a new flag (ie. --identify) that would be required to enable instance identification. This would not add any overhead to gunicorn use cases that have no reason to employ instance identification.